src/main/java/com/squarepolka/readyci/configuration/AndroidPropConstants.java

```diff
@@ -19,5 +18,6 @@ private AndroidPropConstants() {
     public static final String BUILD_PROP_HOCKEYAPP_TOKEN = "hockappToken";
     public static final String BUILD_PROP_HOCKEYAPP_RELEASE_TAGS = "hockeyappReleaseTags";
     public static final String BUILD_PROP_HOCKEYAPP_RELEASE_NOTES = "hockeyappReleaseNotes";
+    public static final String BUILD_PROP_FILENAME_FILTERS = "fileNameFilters";
```

Comment:
Hey @timmutton , do you mind adding some documentation for the new fileNameFilters parameter on the Github wiki and example config file?